backend/windmill-common/src/cache.rs

```diff
@@ -0,0 +1,136 @@
+use quick_cache::sync::Cache;
+use sqlx::{PgExecutor, types::{Json, JsonRawValue as RawValue}};
+
+/// Cache directory for windmill server/worker(s).
+pub const CACHE_DIR: &str = "/tmp/windmill/cache/";
+
+pub mod flow {
+    use super::*;
+    use crate::{error, flows::FlowNodeId};
+
+    /// Cache directory for windmill server/worker(s) flow nodes.
+    pub const CACHE_DIR: &str = const_format::concatcp!(super::CACHE_DIR, "flow");
+
+    lazy_static::lazy_static! {
+        /// Flow node cache.
+        /// FIXME: This should be a static but [`Cache`] does not have a static constructor.
+        pub static ref CACHE: Cache<FlowNodeId, Val> = Cache::new(1000);
+    }
+
+    /// Flow node cache value.
+    #[derive(Debug, Clone, Default)]
+    pub struct Val {
```

Comment:
maybe we will just write a second cache but we will likely reuse same-tiered caching for normal workspace scripts as well so might make sense to generalize with a type arg?